dbms/include/DB/Columns/ColumnVector.h

```diff
@@ -265,6 +265,16 @@ class ColumnVector final : public IColumn
 		res = typename NearestFieldType<T>::Type(data[n]);
 	}
 
+	const T & get(size_t n) const
```

Comment:
Existence of two different `get` methods, one virtual and another not, is a mess.